geowave-vector/src/main/java/mil/nga/giat/geowave/vector/plugin/lock/AuthorizedLock.java

```diff
@@ -18,6 +19,9 @@
  * 
  * 
  */
+@SuppressFBWarnings({
+	"SE_TRANSIENT_FIELD_NOT_RESTORED"

```

Comment:
@rwgdrummer 
Comment call it out (after serialization/deserialization setlockmanager needs to be called), but is that enforced in any way;  (i.e. will geotools serialize this class somehow - and if so, do we ensure/test that we can cause setlockmanager to be called?